app/components/layout/nav.tsx

```diff
@@ -74,9 +97,9 @@ export function Nav() {
                     className="xl:text-md inline-flex items-center px-1 pt-1 text-xs font-medium 2xl:text-lg"
                   >
                     <span
-                      className={`flex h-full w-full items-center ${
-                        pathName === item.to ? "font-semibold" : "font-normal"
-                      } text-primary hover:font-semibold`}
+                      className={`flex h-full w-full items-center text-[#757575] ${
+                        pathName === item.to ? "font-semibold text-primary" : "font-normal"
+                      }  hover:font-semibold hover:text-primary`}
```

Comment:
lets remove the `hover:font-semibold` because it causes the text length to be not consistent when hovering the nav

Reply:
finish mas